.merlin

```diff
@@ -1,4 +1,4 @@
-PKG ppx_deriving ppx_tools rpclib ppx_tools.metaquot result cow cmdliner rresult lwt async oUnit ppx_deriving_rpc
+PKG ppx_deriving ppx_tools rpclib ppx_tools.metaquot result cow cmdliner rresult lwt async oUnit ppx_deriving_rpc cppo_ocamlbuild
```

Comment:
We can but I will get rid of it once we port it to `jbuilder`, I did not spend time on doing it.